src/Helper/InternalHelper.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace PHPSu\Helper;
+
+use PHPSu\Controller;
+use PHPSu\Exceptions\CommandExecutionException;
+use PHPSu\Exceptions\EnvironmentException;
+use PHPSu\Process\CommandExecutor;
+use PHPSu\Tools\EnvironmentUtility;
+
+final class InternalHelper
+{
+
+    public function getCurrentPHPSUVersion(): string
+    {
+        if ($this->isVendorPackage()) {
+            return $this->getPhpSuVersionFromVendor();
+        }
+        if ((new EnvironmentUtility)->isGitInstalled()) {
+            return trim($this->getPhpSuVersionFromGitCommand());
+        }
+        if ($this->isGitFolderAvailable()) {
+            return $this->getPhpSuVersionFromGitFolder();
+        }
+        @trigger_error('Could not detect phpsu-version properly - using development instead', E_USER_NOTICE);
```

Comment:
should we really trigger this E_USER_NOTICE?

The user can notice it by running the ``list`` command and looking at the version.